nRF Toolbox/Profiles/BloodPressure/BloodPressureTableViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BloodPressureTableViewController.swift
+//  nRF Toolbox
+//
+//  Created by Nick Kibysh on 01/10/2019.
+//  Copyright © 2019 Nordic Semiconductor. All rights reserved.
+//
+
+import UIKit
+import CoreBluetooth
+
+extension Identifier where Value == Section {
+    static let bloodPressure: Identifier<Section> = "bloodPressure"
+    static let cuffPressure: Identifier<Section> = "cuffPressure"
+    static let dateTime: Identifier<Section> = "TimeBloodPressureSection"
+    static let heartRate: Identifier<Section> = "HeartRate"
+}
+
+class BloodPressureTableViewController: PeripheralTableViewController {
+    private var bloodPressureSection = BloodPressureSection(id: .bloodPressure)
+    private var cuffPressureSection = CuffPressureSection(id: .cuffPressure)
+    private var dateTimeSection = TimeBloodPressureSection(id: .dateTime)
+    private var heartRateSection = PulseBloodPressureSection(id: .heartRate)
+    
+    override var internalSections: [Section] { [bloodPressureSection, heartRateSection, dateTimeSection, cuffPressureSection] }
+    override var peripheralDescription: Peripheral { .bloodPressure }
+    
+    private var dataSectionIds: [Identifier<Section>] = [.bloodPressure, .cuffPressure]
+    override var navigationTitle: String { "Blood Pressure Monitor" }
+    
+    override func didUpdateValue(for characteristic: CBCharacteristic) {
+        switch characteristic.uuid {
+        case CBUUID.Characteristics.BloodPressure.measurement:
+            let bloodPressureCharacteristic = BloodPreasureCharacteristic(data: characteristic.value!)
+
+            bloodPressureSection.update(with: bloodPressureCharacteristic)
+            heartRateSection.update(with: bloodPressureCharacteristic)
+            dateTimeSection.update(with: bloodPressureCharacteristic)
+            
+            setBloodPressureVisibility(true)
+            cuffPressureSection.isHidden = true
+            tableView.reloadData()
+        case CBUUID.Characteristics.BloodPressure.intermediateCuff:
+            cuffPressureSection.update(with: CuffPreasureCharacteristic(data: characteristic.value!))
```

Comment:
Can we use `guard let value = characteristic.value` instead of force-unwrapping?

Reply:
I'm also force unwrapping when the value must be there. But after reading a characteristic it doesn't need to be there, assuming you connect to something that pretends to be a working device (like nRF Connect). I don't know the whole context here.